README.md

```diff
@@ -1,163 +1,121 @@
 # ZzHorizontalProgressBar
```

Comment:
Add the build & quality badges

Reply:
we will resolve this after the project run on master branch